extension/src/test/Xliff/XliffCache.test.ts

```diff
@@ -0,0 +1,55 @@
+import { xliffCache } from "../../Xliff/XLIFFCache";
+import * as path from "path";
+import { Xliff } from "../../Xliff/XLIFFDocument";
+import * as assert from "assert";
+
+suite("XlfCache Tests", () => {
```

Comment:
Suggested test: onDidChangeTextDocument
Do a text edit in a xlf file and save it, it should update thet cache

Reply:
Added :+1: